packages/frontend/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import {
+  Wrapper,
+  ImageContainer,
+  RecipeName,
+  BottomPanel,
+  RatingWrapper,
+  Rating,
+  StarIcon,
+  HeartIcon,
+  Author,
+} from "../Card/Card.styled";
+import starIcon from "src/assets/images/star.svg";
+import heartIcon from "src/assets/images/heart.svg";
+import { Routes } from "src/enums/routes.enum";
+import { CardProps } from "./Card.types";
+
+export const Card: React.FC<CardProps> = ({ id, photoUrl, title, rating, total, author }) => {
+  const [isFilled, setIsFilled] = useState(false);
```

Comment:
to nie powinine być stan tego komponentu, ten komponent powinien przyjmować tylko propy i nie powinny być w nim zdefiniowane żadne metody